recipes/Polygon3/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "Polygon3" %}
+{% set version = "3.0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Polygon3-{{ version }}.tar.gz
+  sha256: 2ddf8d06975f728d5b40786136c82e5b9d38a846bce236b7e6587bbd6a5e9b49
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - Polygon
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://www.j-raedler.de/projects/polygon
+  summary: Polygon3 is a Python-3 package that handles polygonal shapes in 2D
+  license: LGPL-2.1-or-later
+  license_file: LICENSE.txt
```

Because you are revendoring GPC, you need to extract the license of the version of GPC that you have and include it here. For example:

```suggestion
  license: LGPL-2.1-or-later AND OTHER
  license_file:
    - LICENSE.txt
    - GPC-licenset.txt
```

---

Hi and thank you for reviewing this. I have added a license file for GPC as you suggested.